docs/osdctl_servicelog_list.md

```
@@ -6,18 +6,21 @@ Get service logs for a given cluster identifier.
 
 Get service logs for a given cluster identifier.
 
-# To return just service logs created by SREs
-osdctl servicelog list --cluster-id=my-cluster-id
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)